src/Compilers/CSharp/Test/Semantic/Semantics/CollectionLiteralTests.cs

```diff
@@ -3818,36 +3819,6 @@ static void Main()
                 expectedOutput: "[1, 2, 3], [1, 2, 3], ");
         }
 
-        [Fact]
-        public void SpreadElement_10()
-        {
-            string source = """
-                class Program
-                {
-                    static void Main(string[] args)
-                    {
-                        int[] a;
-                        bool b = args.Length > 0;
-                        a = [..[]];
```

Comment:
I wasn't sure why the test was deleted. These seem like good scenarios.

Reply:
It looked like this test was redundant with `_05` and `_06`. I've added another case to `_05`.